include/wil/winrt.h

```diff
@@ -439,9 +430,9 @@ namespace wil
             struct type // T holder
             {
                 type() {};
-                type(T&& value) : m_value(wistd::forward<T>(value)) {};
+                type(T&& value) WI_NOEXCEPT : m_value(wistd::forward<T>(value)) {};

```

Comment:
Anything storing a `T` should not have its copy/move constructors/assignment operators explicitly marked as `noexcept`. It should, at best, conditionally declare them as `noexcept`